### ISSUE TITLE
Plugin crashes on calling method of imported domain

### ISSUE THREAD

**Issue by eaglus** [effector/swc-plugin-legacy#25]
```ts
import { rootDomain } from "./rootDomain";

const subDomain = rootDomain.store('subDomain');
```
thread 'fixture_tests__fixtures__external_root_domain__code_js' panicked at 'called `Option::unwrap()` on a `None` value', src/effector.rs:320:28

because there are no self.state.loc

This code in my pull request fixes a bug:
https://github.com/effector/swc-plugin/pull/22/files#r1082336331

**Comment by sergeysova** [comment]
@Drevoed Is it fixed by #22?

**Comment by Drevoed** [comment]
Yup!

Fixed by #22.
